lib/arke/exchange/base.rb

```diff
@@ -17,5 +17,23 @@ def initialize(opts)
       rate_limit = 1.0 if rate_limit <= 0
       @min_delay = 1.0 / rate_limit
     end
+
+    def build_error(response)
```

Comment:
this helper don't make much sense, also parsing json twice

Reply:
I've just copypasted it from `Rubykube` exchange, I can redo